site/en/docs/privacy-sandbox/fedcm/index.md

```diff
@@ -216,26 +244,547 @@ They can do the same for Chrome on desktop by going to
+Host: idp.example
+Referer: https://rp.example/
+Accept: application/json
+Sec-Fetch-Dest: webidentity
+```
+
+The properties for the client metadata endpoint include:
+
+<table class="with-heading-tint with-borders">
+  <thead>
+    <tr>
+      <th>Property</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tr>
+    <td><code>privacy_policy_url</code> (optional)</td>
+    <td>RP privacy policy URL.</td>
+  </tr>
+  <tr>
+     <td><code>terms_of_service_url</code> (optional)</td>
+     <td>RP terms of service URL.</td>
+  </tr>
+</table>
+
+The browser expects a JSON response from the endpoint:
+
+```json
+{
+  "privacy_policy_url": "https://rp.example/privacy_policy.html",
+  "terms_of_service_url": "https://rp.example/terms_of_service.html",
+}
+```
+
+The returned client metadata is consumed by the browser and will not be
+available to the RP.
+
+#### Accounts list endpoint {: #accounts-list-endpoint }
+
+The IdP's accounts list endpoint returns a list of accounts that the user is
+currently signed in on the IdP. If the IdP supports multiple accounts, this
+endpoint will return all signed in accounts.
+
+The browser sends a `GET` request with cookies, but without a `client_id`
+parameter and the `Referer` header. This effectively prevents the IdP from
+learning which RP the user is trying to sign in to. For example:
+
+```http
+GET /accounts.php HTTP/1.1
+Host: idp.example
+Accept: application/json
+Cookie: 0x23223
+Sec-Fetch-Dest: webidentity
+```
+
+The browser expects a JSON response that includes an `accounts` property
+with an array of account information with following properties:
+
+<table class="with-heading-tint with-borders">
+  <thead>
+    <tr>
+      <th>Property</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tr>
+    <td><code>id</code> (required)</td>
+    <td>Unique ID of the user.</td>
+  </tr>
+  <tr>
+    <td><code>name</code> (required)</td>
+    <td>Given and family name of the user.</td>
+  </tr>
+  <tr>
+    <td><code>email</code> (required)</td>
+    <td>Email address of the user.</td>
+  </tr>
+  <tr>
+    <td><code>given_name</code> (optional)</td>
+    <td>Given name of the user.</td>
+  </tr>
+  <tr>
+    <td><code>picture</code> (optional)</td>
+    <td>URL of the user avatar image.</td>
+  </tr>
+  <tr>
+    <td><code>approved_clients</code> (optional)</td>
+    <td>An array of RP client IDs which the user has registered with.</td>
+  </tr>
+</table>
+
+Example response body:
+
+```json
+{
+ "accounts": [{
+   "id": "1234",
+   "given_name": "John",
+   "name": "John Doe",
+   "email": "john_doe@idp.example",
+   "picture": "https://idp.example/profile/123",
+   "approved_clients": ["123", "456", "789"],
+  }, {
+   "id": "5678",
+   "given_name": "Johnny",
+   "name": "Johnny",
+   "email": "johnny@idp.example",
+   "picture": "https://idp.example/profile/456"
+   "approved_clients": ["abc", "def", "ghi"],
+  }]
+}
+```
+
+If the user is not signed in, respond with HTTP 401 (Unauthorized).
```

Comment:
Do we need to specify the response error?

Reply:
I had this conversation with @samuelgoto and agreed with this error code.